src/styles/main.scss

```diff
@@ -1,7 +1,78 @@
-@import "utils";
-@import "fonts";
-@import "typography";
+@import "./utils/variables";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
 
 body {
-  background: $c-gray;
+  font-family: "Poppins", sans-serif;
+  background-color: $color-black;
 }
+
+.main {
+  display: grid;
+  gap: 80px;
+  padding: 60px 20px;
+
+  max-width: 1260px;
+  margin: 0 auto;
+
+  &__section {
+    &-title {
+      color: $color-white;
+      font-size: var(--h2-size);
+    }
+  }
+
+  @media (min-width: $media-tablet) {
+    gap: 120px;
+    padding: 120px 72px;
+  }
+
+  @media (min-width: $media-desktop) {
+    padding: 120px;
+  }
+}
+
+button {
+  cursor: pointer;
+}
+
+.failed {
```

Comment:
Maybe, let's make this class a BEM modifier for inputs?

Reply:
im not sure how to do this, bcs this class is added by JS and it is for contact__input and contact__textarea, I can compicate js but i dont think it is worth

Reply:
Okay.